core/main.ts

```diff
@@ -102,3 +104,53 @@ export function main(coreExecutionRequest: Uint8Array | string): Uint8Array | st
 
   return dataform.CoreExecutionResponse.encode(coreExecutionResponse).finish();
 }
+
+function loadActionConfigs(session: Session, filePaths: string[]) {
+  filePaths
+    .filter(path => path.startsWith(`definitions${utils.pathSeperator}`))
+    .filter(path => path.endsWith("actions.yaml"))
```

Comment:
Why two filters instead of `path.startsWith && path.endsWith`?

Reply:
Done.